src/query/storages/stage/src/file.rs

```diff
@@ -29,11 +27,11 @@ use tracing::warn;
 use crate::StageTable;
 
 pub async fn stat_file(
-    table_ctx: Arc<dyn TableContext>,
+    dop: &DataOperator,
```

Comment:
After `StageTable::get_op` has been refactored, we can pass an `Operator` here directly.